routes/products.js

```diff
@@ -2,8 +2,33 @@ const express = require("express");
 const router = express.Router();
 const Products = require("../Products.json");
 
-router.get("/", (req, res) => {
-  res.send(Products);
+router.get("/", async(req, res) => {
+  try{
+
+  
+    const {minPrice,maxPrice}=req.body
```

Comment:
Use a get request get the minPrice and maxPrice by the query string instead of post request

Reply:
ok

Reply:
Now I added the get request and now we can use query string to get the products bt minPrice and maxPrice.

Now the URL string looks like [ http://localhost:5000/products?minPrice=10&maxPrice=18 ] and it returns all products available bt this price.